gui-tests/src/test/java/org/jboss/tools/intellij/quarkus/BasicTests.java

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.intellij.quarkus;
+
+import com.intellij.remoterobot.RemoteRobot;
+import org.jboss.tools.intellij.quarkus.fixtures.dialogs.WelcomeFrameDialogFixture;
+import org.jboss.tools.intellij.quarkus.utils.BuildUtils;
+import org.jboss.tools.intellij.quarkus.utils.GlobalIdeUtils;
+import org.jboss.tools.intellij.quarkus.utils.ProjectToolWindowUtils;
+import org.jboss.tools.intellij.quarkus.utils.quarkus.Utils;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static com.intellij.remoterobot.stepsProcessing.StepWorkerKt.step;
+
+
```

Comment:
There should be also author and also short description what it does.

Reply:
I have added author and comments to every Java file.